xbmc/Application.cpp

```diff
@@ -2734,11 +2711,42 @@ void CApplication::OnApplicationMessage(ThreadMessage* pMsg)
   break;
 
   case TMSG_LOADPROFILE:
-  {
     CGUIWindowLoginScreen::LoadProfile(pMsg->param1);
     break;
+
+  default:
+    CLog::Log(LOGERROR, "%s: Unhandled threadmessage sent, %u", __FUNCTION__, pMsg->dwMessage);
```

Comment:
@tamland If we end up here someone seriously screwed something up, I'm thinking maybe we should add an assert() to catch things early. I know I don't go through the logs that often.